_posts/2023-11-03-this-week-in-rails.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: '"INSERT...RETURNING" for MariaDB, "SET CONSTRAINTS" for PostgreSQL, "DEFERRABLE" foreign keys for SQLite and much more!'
+categories: news
+author: zzak
+published: true
+date: 2023-11-03
+---
+
+
+Hey, [zzak](https://github.com/zzak) here with the first edition of This Week in Rails for November, 2023.  
+
+[Support RETURNING clause for MariaDB](https://github.com/rails/rails/pull/49840)  
```

Comment:
Heading and content need a space between for formatting I think- 
```suggestion
[Support RETURNING clause for MariaDB](https://github.com/rails/rails/pull/49840)  

```

Reply:
@vipulnsward I believe there is no space needed but 2 "empty spaces" at the end of line to have it behave like "br" tag

Reply:
<img width="760" alt="Screenshot 2023-11-03 at 19 07 46" src="https://github.com/rails/website/assets/277819/893fb87b-bac4-449d-8064-9d4dcfd79499">

Here is the rendered version.

I use the double space to insert a line break, it doesn't show up in the diff but if you highlight the line it's there.